### PR TITLE
add Softsign layer

### PR DESCRIPTION
**Pull request by nishnik**
Softsign activation function.

Ref: [theano implementaion](https://github.com/Theano/theano/blob/52903f8267cff316fc669e207eac4e2ecae952a6/theano/tensor/nnet/nnet.py#L2428)

### CODE REVIEW

File: test/test_serialization.h [hunk 1]
```diff
@@ -739,6 +739,30 @@ TEST(serialization, serialize_softplus) {
...
+                    "width": 10,
+                    "height" : 10,
+                    "depth" : 3
+                }
+            }
+        ]
+    }
+    )";
+
+  net.from_json(json);
```
Comment (nishnik) [inline test/test_serialization.h]: There is an error at this line but I don't knoe how to fix it (Possibly a problem with json?). Any suggestion @karandesai-96 @Randl ?

Comment (edgarriba) [inline test/test_serialization.h]: @nishnik are your tests passing in local?

Comment (nishnik) [inline test/test_serialization.h]: No! but I can not figure out why

Comment (edgarriba) [inline test/test_serialization.h]: maybe a corrupted copy paste? try to rewrite it by yourself

File: tiny_dnn/util/serialization_layer_list.h [hunk 2]
```diff
@@ -44,6 +44,7 @@ void register_layers(T* h) {
   h->template register_layer<elu_layer>("elu");
   h->template register_layer<tanh_p1m2_layer>("tanh_scaled");
   h->template register_layer<softplus_layer>("softplus");
+  h->template register_layer<softplus_layer>("softsign");
```
Comment (nyanp) [inline tiny_dnn/util/serialization_layer_list.h]: @nishnik
Thx to your contribution ;)
s/softplus/softsign/
This might be the cause of the error.

Comment (nishnik) [inline tiny_dnn/util/serialization_layer_list.h]: Thanks! It works.

**Comment by nyanp** [comment]
LGTM 👍

**Pull request merged**